watertap/examples/flowsheets/case_studies/activated_sludge/ASM1_flowsheet.py

```diff
@@ -0,0 +1,364 @@
+#################################################################################
+# The Institute for the Design of Advanced Energy Systems Integrated Platform
+# Framework (IDAES IP) was produced under the DOE Institute for the
+# Design of Advanced Energy Systems (IDAES), and is copyright (c) 2018-2021
+# by the software owners: The Regents of the University of California, through
+# Lawrence Berkeley National Laboratory,  National Technology & Engineering
+# Solutions of Sandia, LLC, Carnegie Mellon University, West Virginia University
+# Research Corporation, et al.  All rights reserved.
+#
+# Please see the files COPYRIGHT.md and LICENSE.md for full copyright and
+# license information.
+#################################################################################
+"""
+Example of activated sludge process model using ASM1.
+
+
+Unit opeations are modeled as follows:
+
+    * Anoixic reactors as standard CSTRs (CSTR)
+    * Aerobic reactors as CSTRs with additioanl injection terms (CSTR_Injection)
```

Comment:
Maybe explicitly say that `S_O` is the injection term

Reply:
@yalinli2 Whilst `S_O` is the only term we will use in the example, there are injection terms for all species.